index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/34795ce01c.js" crossorigin="anonymous"></script>
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav>
+        <header class="logo"><a href="#" target="_blank">My Logo</a></header>
+        <i class="fa-solid fa-bars"></i>
+    </nav>
+    <section class="headline">
+        <img src="header-shapes mobile.png" alt="shapes">
+        <h1 class="headline-title">I'm Dhiraj<br>Glad to see<br>you!</h1>
+        <p class="headline-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="headline-connect">LET’S CONNECT</p>
```

Comment:
- [ ] Kindly take note that this suppose to be a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design). Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href attribute. 😄

Reply:
Thank's for the feedback.
I wanted to confirm do I need to change <h1> and both <p> text into a link <a>/<button> text.

Reply:
@OyePriscilla I have added a link tag for the LET'S CONNECT text and included styling: Hover. Also, the README file is filled in with more details about the project.
I appreciate any feedback.😁

Reply:
@dhirajsingha143 Awesome!!!, well-done 👍